content/docs/tutorials/interactivity.md

```diff
@@ -0,0 +1,26 @@
+---
+$title: Create interactive AMP pages
```

Comment:
Nit: Other tutorials capitalize every word, i.e. "Create Interactive AMP Pages".

Also I used the "advanced" qualifier in the codelab name to emphasize that AMP pages can already be interactive without amp-bind. I'm fine with the current name -- just food for thought.